mantidproject/mantid
# Python in mantid

**Description of work.**

Update the Current Python in Mantid course by moving it to the user docs and making it accurate for Workbench

**To test:**
Build docs.
* Check page order, links and formatting.
* Check I have had the courses and solutions checked by other devs!
* Check all images and plot directives appear
* Check the exercises
* Check the plot script generation in Workbench gives the link to the scripting_plots webpage (moved from `docs/source/plotting/scripting_plots` to `docs/source/tutorials/python_in_mantid/plotting/02_scripting_plots`)

Is this fixed? https://forum.mantidproject.org/t/integration-algorithm-not-giving-sensible-or-reproducible-results/666/6
Fixes #28509
Fixes #25796 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
